data/json/items/tools.json

```diff
@@ -6193,6 +6193,22 @@
     }
   },
   {
+    "id": "hand_drill",
+    "type": "TOOL",
+    "symbol": ";",
+    "color": "light_gray",
+    "name": "hand drill",
+    "description": "A primitive manual drill with a single drill bit.  It is slow and it will exhaust you quickly.",
```

Comment:
So far it's just text. What do you think of adding a flag for exhausting tool and burning some stamina on use if the flag is present?